src/docfx/lib/json/SourceInfoJsonConverter.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Docs.Build
+{
+    internal class SourceInfoJsonConverter : JsonConverter
```

Comment:
Separate other converter and contract resolver of `JsonUtility` into an isolated file as well?

Reply:
do it in another PR